index.html

```diff
@@ -155,11 +155,11 @@ <h3>Skills</h3>
 
         
        
-          <form action="">
+          <form action="https://formspree.io/f/xdojqgbe" method="post">
            
-           <input type="text" id="name" name="name" placeholder="Full name">
-            <input type="text" id="email" name="email" placeholder="Email Address">
-            <textarea name="comments" id="comments" cols="30" rows="10">Enter text here</textarea>
+           <input type="text" id="name" name="name" placeholder="Full name" minlength="30" required>
```

Comment:
- [ ] Kindly make sure on `input:name` have the `maxlength` attribute as required. I think you are mistaken to use the `minlength` attribute instead of the `maxlength`. Please implement that to meet the requirements :wink: